app/controllers/suppress_collections_controller.rb

```diff
@@ -5,13 +5,15 @@ class SuppressCollectionsController < ApplicationController
   respond_to :html, :json
 
   def index
-    @response = RestClient.get("#{fetch_env_vars['API_HOST']}/harvester/sources", { params: {:"source[status]" => "suppressed", api_key: fetch_env_vars['HARVESTER_API_KEY'] } })
-    @blacklisted_sources = JSON.parse(@response)
+    @blacklisted_response = RestClient.get("#{fetch_env_vars['API_HOST']}/harvester/sources", { params: {:"source[status]" => "suppressed", api_key: fetch_env_vars['HARVESTER_API_KEY'] } })
```

Comment:
can we put some line breaks in on the hash or something?  This is beast.

Reply:
or declare a params hash each time, and pass it to rest client

Reply:
done - I've broken down into options variables - thanks